nncf/torch/sparsity/collector.py

```diff
@@ -71,7 +71,8 @@ def _collect_weights_descriptions(self) -> List[WeightDescription]:
             processed_modules.append(minfo.module)
 
         compression_types = tuple(COMPRESSION_MODULES.registry_dict.values())
-        for module_name, module in self._model.get_nncf_wrapped_model().named_modules():
+        modules_to_process = {k: v for k, v in self._model.named_modules() if not k.startswith('_nncf')}
```

Comment:
Is this safe?

Reply:
Safe in what sense specifically? This approach does implicitly mean that we should not store sparsity-processed modules in the `NNCFNetworkInterface` namespace, but we don't do that now and I don't see when we will need to. The possibility that the user's module would start with `_nncf` is quite low.